docs/README.md

```diff
@@ -1,69 +1,89 @@
-# Azure Plugins
-<!--
-  Include a short overview about the plugin.
-
-  This document is a great location for creating a table of contents for each
-  of the components the plugin may provide. This document should load automatically
-  when navigating to the docs directory for a plugin.
-
--->
+The Azure plugin can be used with HashiCorp Packer to create custom images on Azure.
+To do so, the plugin exposes multiple builders, among which you can choose the one most adapted to your workflow.
 
 ## Installation
 
-### Using pre-built releases
-
-#### Using the `packer init` command
-
-Starting from version 1.7, Packer supports a new `packer init` command allowing
-automatic installation of Packer plugins. Read the
-[Packer documentation](https://www.packer.io/docs/commands/init) for more information.
-
-To install this plugin, copy and paste this code into your Packer configuration .
-Then, run [`packer init`](https://www.packer.io/docs/commands/init).
+To install this plugin, copy and paste this code into your Packer configuration, then run [`packer init`](https://www.packer.io/docs/commands/init).
 
 ```hcl
 packer {
   required_plugins {
     azure = {
-      version = ">= 1.4.2"
       source  = "github.com/hashicorp/azure"
+      version = "~> 1"
```

Comment:
Now that v2 is out, we could update this block to point to this version I would think

Reply:
Updated.